cli/tests/unit/fetch_test.ts

```diff
@@ -221,17 +221,6 @@ unitTest({ perms: { net: true } }, async function responseClone(): Promise<
   }
 });
 
-unitTest({ perms: { net: true } }, async function fetchEmptyInvalid(): Promise<
-  void
-> {
-  await assertThrowsAsync(
-    async () => {
-      await fetch("");
-    },
-    URIError,
-  );
-});
```

Comment:
We should still run this test when `--location` is not present. Might have to add `location` option to `unitTest` definition 🤔  could you turn it into integration test in the mean time?